.gitignore

```diff
@@ -11,3 +11,5 @@ docs/reference.md
 coverage-report/
 
 bench.json
+
+tests/file-tests/
```

Comment:
I've been working on implementing this and I had a quick query: 

`uiop:setup-temporary-directory` and `uiop:temporary-directory` both call `uiop:default-temporary-directory`, which provides *the* default temporary directory, an environment-specific directory which can hold other temp files from the system. Each time it will be the same directory. This seems to be contrary to the idea of temporary directories in Rust, Python, Haskell, etc.

In my original thinking, I had assumed the coalton function, `create-temporary-directory` would generate a unique, empty directory, which could be used by `with-temporary-directory` to perform an action before recursively removing the directory.

I have already implemented `create-temporary-file` as a unique file with a given extension in the `~/tmp/` directory. Does it make the most sense to prefix temp files and temp directories with either `~/tmp/` or the directory returned by `uiop:temporary-directory` above?

Reply:
Prefix with `(uiop:temporary-directory)`. `~/tmp` isn't a temporary directory on any system I know.